AndelaOSP/Andela-Socials
# Feature #159402806: Create upload component UI only

#### What Does This PR Do?

Implement The UI for the Upload component
- [x] Display image to upload after selection
- [x] Implement UI design for the Upload Component
- [x] Add clear uploads button
- [x] Handle invalid file formats

#### How can this be manually tested?

Assuming you've followed the [README.md](https://github.com/AndelaOSP/Andela-Socials/blob/develop/README.md) to setup the project

- Run scripts/start.sh
- Navigate to `localhost:9000` and login using your email(gmail)
- You should be able to click the `Add Event` button and play with the Upload Component

#### What are the relevant github issues?

[#159402806](https://www.pivotaltracker.com/n/projects/2174978/stories/159402806)

#### Screenshot/Demo

![demo_ui_upload](https://user-images.githubusercontent.com/19430799/45170887-9f306980-b209-11e8-900b-474591f82561.gif)
